src/main/java/reposense/report/ReportGenerator.java

```diff
@@ -91,7 +91,8 @@ public static void updateRepoConfig(RepoConfiguration config) {
             logger.warning(String.format("%s/%s/%s is malformed.",
                     config.getDisplayName(), REPOSENSE_CONFIG_FOLDER, REPOSENSE_CONFIG_FILE));
         } catch (FileNotFoundException fnfe) {
-            throw new AssertionError("This should not happen.");
+            throw new AssertionError(
+                    "FileNotFoundException should not happen as path has been checked to exists");
```

Comment:
`FileNotFoundException should not happen as path has been checked to exists`

=>

`This exception should not happen as we have performed the file existence check.`